### ISSUE TITLE
Mac M1 - ARM - LibLiteCore not found

### ISSUE THREAD

**Issue by SchroterQuentin** [couchbase/couchbase-lite-net#1549]
### Library Version

3.1.0

### .NET Runtime

dotnet 7.0

### Operating System / Device Details

Mac M1

### Log Output

> Unhandler exception. System.TypeInitializationException: The type initializer for 'Couchbase.Lite.Internal.Logging.WriteLog' threw an exception.
> ---> System.DllNotFoundException: Unable to load shared library 'LiteCore' or one of its dependencies. In order to help diagnose loading problems, consider setting the LD_DEBUG environment variable: libLiteCore: cannot open shared object file: No such file or directory

### Expected behavior

It should not raise exception ^^.

### Actual behavior

It fails and the application crash.

### Steps To Reproduce

1. Get a MacBook Pro with M1 chip
2. dotnet run an application using CouchbaseLite

### Comments

I remember having the same issue when trying to run CouchbaseLite 3 on raspberry and didn't succeed. I guess it's very similar but this time it prevents me from developping my application on my new computer and all my team is getting new M1 so we won't be able to update our application soon.

Couchbase Server seems to work fine with M1 and according to the documentation Couchbase Lite C is working with Apple M1 too. So could you tell me what I need to do to get this working on my new mac ?

**Comment by borrrden** [comment]
Raspberry pi is a known unsupported scenario, but M1 should work to my knowledge.  I don't actually have an M1 to try but I did have a colleague try with 3.1.0 on an M1 with no issue.  I just confirmed that he is indeed running the arm64 dotnet runtime.  The error message means that either libLiteCore.dylib can't be found for some reason, or one of its dependencies.  Does running `otool -L` on it show anything strange?
